WordPress/WordPressTodayWidget/TodayViewController.swift

```diff
@@ -152,24 +171,51 @@ private extension TodayViewController {
         let data = TodayWidgetStats.loadSavedData()
         visitorCount = data.visitors
         viewCount = data.views
+        likeCount = data.likes
+        commentCount = data.comments
+    }
+
+    func initRows() {
+        guard let row = Bundle.main.loadNibNamed(Constants.rowNibName, owner: nil, options: nil)?.first as? TwoColumnRow else {
+            return
+        }
+
+        row.configure(leftColumnName: LocalizedText.views,
+                      leftColumnData: Constants.noDataLabel,
+                      rightColumnName: LocalizedText.visitors,
+                      rightColumnData: Constants.noDataLabel)
+
+        rowsStackView.addArrangedSubview(row)
+    }
+
+    func configureColors() {
+        configureLabel.textColor = .text
+        configureButton.backgroundColor = .neutral(.shade10)
+        configureButton.setTitleColor(.text, for: .normal)
+
+        // In dark mode, let the separator line be the same color as the label text.
+        separatorLine.backgroundColor = UIColor(light: .divider, dark: .textSubtle)
+        siteNameLabel.textColor = .textSubtle
+        siteUrlLabel.textColor = .textSubtle
     }
 
     func updateLabels() {
-        let numberFormatter = NumberFormatter()
-        numberFormatter.numberStyle = .decimal
-        visitorsCountLabel.text = numberFormatter.string(from: NSNumber(value: visitorCount)) ?? "0"
-        viewsCountLabel.text = numberFormatter.string(from: NSNumber(value: viewCount)) ?? "0"
 
         siteNameLabel.text = siteName
+        siteUrlLabel.text = siteUrl
+
+        guard let row = rowsStackView.arrangedSubviews.first as? TwoColumnRow else {
+            return
+        }
+
+        row.updateData(leftColumnData: displayString(for: viewCount),
+                       rightColumnData: displayString(for: visitorCount))
     }
 
-    func changeTextColor() {
-        configureMeLabel.textColor = .text
-        siteNameLabel.textColor = .text
-        visitorsCountLabel.textColor = .text
-        viewsCountLabel.textColor = .text
-        visitorsLabel.textColor = .textSubtle
-        viewsLabel.textColor = .textSubtle
+    func displayString(for value: Int) -> String {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
```

Comment:
Can the formatter be a `let` stored property with closure? Just to avoid to create a formatter every time the method is invoked.